website/content/installation/_index.md

```diff
@@ -206,7 +205,9 @@ named `metallb-operator`:
 kubectl edit csv metallb-operator
 ```
 
-and change the `BGP_TYPE` environment variable of the `manager` container to `frr`:
+{{% /notice %}}
```

Comment:
This would actually split the note in 2, cutting the text. Not sure it's what we want